distribution/docs/src/main/resources/content/_configuring/no-idp.adoc

```diff
@@ -7,10 +7,16 @@
 
 == {title}
 
-To configure ${branding} to not use an Identity Provider (IdP), simply disable the IdP feature.
-
-. Navigate to the *${admin-console}*.
-. Select the *System* tab.
-. Select the *Features* tab.
-. Uninstall the `security-idp` feature.
+To configure ${branding} to not use an Identity Provider (IdP),
```

Comment:
✏️ This is super petty and I honestly consider this to be overly pedantic and ofttimes it reduces clarity but...
`To configure ${branding} not to use...`

Reply:
But really...you shouldn't change that. I think it's better _to boldly go where_ lots of docs have gone before.